<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BioDevicesWidget</class>
 <widget class="QWidget" name="BioDevicesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>331</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Please select other biometric devices</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblDeviceType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Device Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDeviceName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>152</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Device Name:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbDeviceName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>321</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <zorder>lblTitle</zorder>
  <zorder>lblDeviceType</zorder>
  <zorder>lblDeviceName</zorder>
  <zorder>cmbDeviceName</zorder>
  <zorder>btnBack</zorder>
  <zorder>btnOK</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
